packages/babel-plugin-transform-flow-comments/src/index.js

```diff
@@ -4,8 +4,21 @@ export default function ({ types: t }) {
     path.replaceWith(t.noop());
   }
 
+  // https://github.com/babel-plugins/babel-plugin-flow-comments/pull/6#issuecomment-122709012
+  function escapeSource(source) {
+    return source.replace(/\*-\//g, "*-ESCAPED/").replace(/\*\//g, "*-/");
+  }
+
+  function getSource(path) {
+    if (path.isClassProperty()) {
+      return (path.node.static ? "static " : "") + path.get("key").getSource() + path.get("typeAnnotation").getSource();
```

Comment:
Having to special-case `ClassProperty` as `path.getSource()` would print out the _value_ of the node too, which we shouldn't duplicate really.

e.g.

```js
class X {
  a = 3; /*:: a: number = 3*/
}
```